umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -25,10 +25,19 @@ The UMH includes 3 pre-configured data simulators for testing connections:
 OPC UA, often complex, can be streamlined using our Benthos-based OPC UA connector
 accessible from the Management Console.
 
+### Visualize Connections or Network Devices
+
+From the **Network View**, you can visualize the connections or network devices a UMH
+instance is connected to. This view provides the latest status of both the instances and
+the network devices, the latter also showing the latency in milliseconds.
```

Comment:
one can also see the status of the networ kdevice. if it is red, then the connection is unhealthy (e.g., too high latency, or not reachable at all).